zoo-css-challenge/style.css

```diff
@@ -191,5 +208,19 @@ footer {
 /*Programs*/
 
 .programs .card {
-  display: block;
+  .program {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+.card {
+  width: 270px;
+  height: 100px;
+}
+
+.section {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
```

Comment:
```suggestion
```
There are no `section` classes in the HTML document so these CSS properties don't change anything, so we should delete this since it's not doing anything